droid-hal-device.inc

```diff
@@ -53,6 +53,7 @@
 #                      from droid-hal-device
 # bootloader_binary: define bootloader binary name when aboot is packaged
 #                      separately from droid-hal-device
+# dtbo_binary:       override dtbo binary name. Defaults to dtbo.img
```

Comment:
I think it should be dtbo_image to fit with the rest.

Reply:
It's binary, because it fits bootloader_binary, they are both images.